targets/targets.json

```diff
@@ -4293,7 +4303,19 @@
         "post_binary_hook": {
             "function": "LPCTargetCode.lpc_patch"
         },
-        "device_name": "LPC54114J256BD64"
+        "device_name": "LPC54114J256BD64",
+        "supported_c_libs": {
+            "arm": [
+                "std", "small"
+            ],
+            "gcc_arm": [
+                "std", "small"
+            ]
+        },
+        "c_lib": "small",
```

Comment:
Same comment as above.

Reply:
The target config "c_lib": "small" should be there for the Mbed 2 target

Reply:
@harmut01 Please remove `"c_lib": "small",`.